docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -4,6 +4,160 @@ sidebar_label: With Argo CD
 
 # Advanced Installation with Argo CD
 
+This section outlines the various advanced installation options available for deploying Kargo on a Kubernetes cluster using Argo CD.
+
+:::warning
+This section assumes that you have already installed any dependencies or prerequisites required for running Kargo on a Kubernetes cluster. Please see the [installation guide](../../40-operator-guide/10-basic-installation.md#prerequisites) for more information.
+:::
+
+## Argo CD Helm Application
+
+The most common way to deploy Kargo using Argo CD is to create a Helm `Application`, and use the `.spec.source.helm.parameters` section to specify any parameters you may need. This is the most straightforward way to deploy Kargo using Argo CD.
+
+:::info
+If using the `api.adminAccount.passwordHash` parameter, you must escape the `$` character with `$$` to prevent Helm from interpreting it as a variable. Please see [this discussion](https://discord.com/channels/1138942074998235187/1138946346217394407/1267966083168469102) for more information.
+:::
+
+```yaml
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: kargo
+  namespace: argocd
+spec:
+  project: default
+  destination:
+    namespace: kargo
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: ghcr.io/akuity/kargo-charts
+    chart: kargo
+    targetRevision: 1.2.0
+    helm:
+      parameters:
+        - name: api.adminAccount.passwordHash
+          value: "$$2a$$10$$Zrhhie4vLz5ygtVSaif6o.qN36jgs6vjtMBdM6yrU1FOeiAAMMxOm"
+        - name: controller.logLevel
+          value: "DEBUG"
+        - name: api.adminAccount.tokenTTL
+          value: "24h"
+        - name: api.adminAccount.tokenSigningKey
+          value: "iwishtowashmyirishwristwatch"
```

Comment:
We need to somehow make it clear that:

* These are _example_ params and that you should use different values if installing anywhere other than a local cluster.
* Any/all installation options can be configured in this manner. A link out to the chart's readme might be in order.

@fykaa did it like this:

https://newdocs.docs.kargo.io/new-docs/operator-guide/advanced-installation/advanced-with-helm